models/officer.js

```diff
@@ -33,6 +33,9 @@ export default sequelize.define('officers', {
     type: DataTypes.DATE,
     defaultValue: null,
   },
+  linkedinUrl: {
+    type: DataTypes.TEXT,
```

Comment:
do you think it would be worth adding a field validator?

Reply:
I’ll validate that it’s a URL - that’s probably all we need. Since officers are the only ones using this, I trust that they’ll put a LinkedIn or similar URL here.